README.md

```diff
@@ -23,9 +23,9 @@ Portfolio: Mobile Version
 v2-desktop-version-with-animations
 Request this url: https://github.com/ogagaoghene/v1-desktop-version-with-animations in your web browser. Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side. Select download Zip option from drop down menu. Once the download is complete you will be able to access my project locally.
 
-## Links 
- https://ogagaoghene.github.io/v1-mobile-web-app/
- 
+## Links (GitHub Page link for application deploy - v1-desktop-version-with-animations)
+https://ogagaoghene.github.io/v1-mobile-web-app/
+
```

Comment:
- [ ] [Optional] It would be better professionally if you add this link in the` live demo` section above in the Readme file.